### DISCUSSION TITLE
Is this use to store in a phone or remote server?

### DISCUSSION THREAD

**Discussion by tony199555** [immich-app/immich#39] [Q&A] [answered]
Hi, everyone. Sorry, maybe I am just dumb. But does this used to store photos in your own phone/other idle phones, or to a remote server?  I try to find out reading the readme a few times but to no avail. Thanks.

**Comment by alextran1502** [comment, marked as answer]
Hey Tony, this application backups/stores the photos/videos on your phone to the remote server in which you are self-hosted, so the data belongs to you and is not exposed to any third-party services.

**Reply by tony199555** [reply]
Awesome. This is what I need. Look forward to PA.
